betago/model.py

```diff
@@ -57,7 +65,7 @@ def next_move():
             json_result = jsonify(**result)
             return json_result
 
-        self.app.run(host='0.0.0.0', debug=True, threaded=True, use_reloader=False)
+        self.app.run(host='0.0.0.0', port=int("8081"), debug=True, threaded=True, use_reloader=False)
```

Comment:
I could use `8080`. I had trouble initially with it in c9 but, it should work (I'm using it right now). I couldn't imagine any reason for `8080` to be blocked, so it should work in nearly any environment (as long as there is no other web server running).